manager/src/test/java/com/redhat/service/smartevents/manager/utils/DatabaseManagerUtils.java

```diff
@@ -62,7 +58,6 @@ public void cleanUpAndInitWithDefaultShard() {
     @Transactional
     public void cleanUp() {
         // Clean up
-        deleteAllWork();
```

Comment:
@manstis Do we need to do anything to clear up the `Quartz` tables at all?

Reply:
See https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/920#discussion_r922037832.

tl;dr; No.